concordium-consensus/src/Concordium/KonsensusV1/TreeState/Implementation.hs

```diff
@@ -301,9 +309,14 @@ mkInitialSkovData ::
     Duration ->
     -- |Bakers at the genesis block
     EpochBakers ->
+    -- |If present the initial skov data created
+    -- will carry over the account nonces and sequence numbers
+    -- from the provided transaction table.
+    -- This is used for migration.
+    Maybe TransactionTable ->
```

Comment:
I'd recommend not complicating things with an optional here, and just require it here, and pass an emptyTransctionTable in a couple of places in tests.

Reply:
That will also simplify `initialiseNewSkovV1` where you don't have to override the transaction table after constructing the state anymore, i.e., no longer need to

```
                    mkInitialSkovData gscRuntimeParameters genMeta pbs genTimeoutDuration genEpochBakers
                        & transactionTable .~ genTT
```

Reply:
Moving over the TT and PT and cleaned up as suggested.